### PR TITLE
React router redux integration

### PR DESCRIPTION
**Pull request by endro42**

### CODE REVIEW

File: src/components/views/buildSomething/BuildSomething.js [hunk 1]
```diff
@@ -31,7 +40,10 @@ class BuildSomething extends Component {
                 LiskHunt has the aim to collect all the resources, categorize
                 and advertise.<br />
                 You can take inspiration on the{' '}
-                <a onClick={goResourcesList.bind(this)} className="underlined">
+                <a
```
Comment (nerdvibe) [inline src/components/views/buildSomething/BuildSomething.js:43]: I think that the " ' " broke Github :D Perhaps we should replace it with `&apos;` ?

Comment (nerdvibe) [reply src/components/views/buildSomething/BuildSomething.js:43]: ( on line 26 )

File: src/components/views/landing/Landing.js [hunk 2]
```diff
@@ -1,34 +1,91 @@
...
+            <p>
+              <span role="img" aria-label="wave">
+                🤓
+              </span>{' '}
+              Are you a developer?{' '}
+              <a
+                className="underlined"
+                onClick={() => this.props.goBuildSomething()}
+              >
+                Let's create stuff →{' '}
```
Comment (nerdvibe) [inline src/components/views/landing/Landing.js]: I believe here happened the same. Let's try to replace `'` with `&apos;`

File: src/components/views/buildSomething/BuildSomething.js (hunk 1 above)
Comment (endro42) [reply src/components/views/buildSomething/BuildSomething.js:43]: Haha do you mean {" "}? This happens because of the prettier i think?

Comment (nerdvibe) [reply src/components/views/buildSomething/BuildSomething.js:43]: I don't think that's the reason, I think the reason is the non escaped " ' " on line 26... let's see what happens if we escape that char :)

File: src/components/views/newHunt/NewHunt.js [hunk 3]
```diff
@@ -5,20 +5,23 @@ import PropTypes from 'prop-types';
...
-      hideManualSubmit: true,
-    };
+  componentWillMount() {
+    this.props.setActivePage(goSubmitHunt);
   }
 
-  togleHideManualSubmit = () => {
-    return this.setState({ hideManualSubmit: !this.state.hideManualSubmit });
+  toggleHideManualSubmit = () => {
+    const manual = this.props.hideManualSubmit == true ? false : true;
```
Comment (nerdvibe) [inline src/components/views/newHunt/NewHunt.js:19]: This could be simplified with `const manual = !this.props.hideManualSubmit`

Comment (endro42) [reply src/components/views/newHunt/NewHunt.js:19]: I don't know why but this was an issue where it did not work with the shortened version. Did it work on your version?

Comment (nerdvibe) [reply src/components/views/newHunt/NewHunt.js:19]: Yes it does... mmmh :shipit:

Comment (nerdvibe) [reply src/components/views/newHunt/NewHunt.js:19]: Anyway, no problem :) It's fixed in my development brach 🚀

**Pull request merged**